app/javascript/packs/ping_thing/stats_chart.js

```diff
@@ -22,9 +22,33 @@ export default {
       interval: 1
     }
   },
+  channels: {
+    PingThingStatChannel: {
+      connected() {
+          console.log("connected to PingThingStats")
```

Comment:
You could comment out this console.logs they might be useful for debugging but shouldn't be used on production.